operator/api/v1alpha2/keptnappversion_test.go

```diff
@@ -25,32 +25,32 @@ func TestKeptnAppVersion(t *testing.T) {
 			PostDeploymentEvaluationStatus: common.StateFailed,
 			WorkloadOverallStatus:          common.StateFailed,
 			Status:                         common.StateFailed,
-			PreDeploymentTaskStatus: []TaskStatus{
+			PreDeploymentTaskStatus: []ItemStatus{
 				{
-					TaskDefinitionName: "defname",
-					Status:             common.StateFailed,
-					TaskName:           "taskname",
+					DefinitionName: "defname",
+					Status:         common.StateFailed,
+					Name:           "taskname",
 				},
 			},
-			PostDeploymentTaskStatus: []TaskStatus{
+			PostDeploymentTaskStatus: []ItemStatus{
 				{
-					TaskDefinitionName: "defname2",
-					Status:             common.StateFailed,
-					TaskName:           "taskname2",
+					DefinitionName: "defname2",
```

Comment:
Not sure if this wouldn't be a breaking change ... @mowies, @thisthat: What do you think?

Reply:
I believe it is a breaking change - so we would need to add a conversion webhook for the KeptnAppVersion CRD

Reply:
Yes, I agree. This is a breaking change. 
I suggest moving this renaming out of this PR because it is not related to a linter problem.

Reply:
Yes, we have 2 breaking changes in here: KeptnAppVersion and KeptnWorkloadInstance
I agree that this is better to do in a separate PR.

Reply:
addressed in a separate PR